src/applications/vaos/appointment-list/components/ScheduleNewAppointment.jsx

```diff
@@ -8,12 +8,16 @@ import {
   selectFeatureRequests,
   selectFeatureStatusImprovement,
   selectFeaturePrintList,
+  selectFeatureStartSchedulingLink,
 } from '../../redux/selectors';
 // eslint-disable-next-line import/no-restricted-paths
 import getNewAppointmentFlow from '../../new-appointment/newAppointmentFlow';
 
 function handleClick(history, dispatch, typeOfCare) {
-  return () => {
+  return e => {
+    // Stop default behavior for anchor tag since we are using React routing.
+    e.preventDefault();
+
     recordEvent({
       event: `${GA_PREFIX}-schedule-appointment-button-clicked`,
     });
```

Comment:
Based on the AC, if the `va_online_scheduling_start_scheduling_link` flag is on, it should record the event `vaos-start-scheduling-link` in UA (Universal Analytics)

Reply:
Sorry, moving to fast!